cmd/argocd/commands/cert.go

```diff
@@ -29,6 +29,20 @@ func NewCertCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 			c.HelpFunc()(c, args)
 			os.Exit(1)
 		},
+		Example: `
+Add a TLS certificate for cd.example.com to ArgoCD cert store from a file
```

Comment:
can you make formatting consistent with other commands?

Reply:
Went over all examples to reformat them. I actually have an idea how to simplify specifying Examples for the CLI. It's currently a real PITA with my IDE (VS Code) to write such kind of texts, because editor keeps replacing two spaces with tabs on every other keystroke. Sigh.